### ISSUE TITLE
Bad mesh quality -  perhaps condition on camera extrinsics?

### ISSUE THREAD

**Issue by frederiknolte** [Stability-AI/stable-fast-3d#64]
Hi, when using images of objects that are not from a canonical pose (e.g. the demo images are always taken from similar elevation as the objects, but this is not guaranteed in general), the reconstruction quality is really bad and objects become essentially flat. 

I see that the transformer network uses default camera extrinsics as a conditioning variable and I was wondering whether you randomised camera position during training such that I can use this variable to improve the mesh quality? Or was it left constant during training?

**Comment by vork** [comment]
Yes the model is trained with random images from random camera poses. You just need to modify the scripts a bit. Check [this section](https://github.com/Stability-AI/stable-fast-3d/blob/main/sf3d/system.py#L267-L272) to see how we construct it. Alternatively, for the gradio demo [here](https://github.com/Stability-AI/stable-fast-3d/blob/main/gradio_app.py#L22-L32).
